### DISCUSSION TITLE
Fast debug

### DISCUSSION THREAD

**Discussion by drozenfa** [reloadware/reloadium#124] [General]
I'm sorry if this is a dumb question but I don't seem to find the answer anywhere. What exactly is Fast Debug and how it differs from Debug?
Thanks!

**Comment by dkrystki** [comment]
Fast Debug is an optimized version of normal debug with some limitations.

![fast_debug_comparison_light](https://user-images.githubusercontent.com/45499236/225653424-0773c4c8-a990-41e6-abc3-3059914a0f80.gif)

I'll explain it a bit more in the documentation soon but basically only functions having breakpoints inside during the start will be debugged in much faster way. Dynamic breakpoints are coming in the future.

It's especially useful when dealing with big python projects having substantially longer startup time in normal debug mode.
Startup time in fast debug mode is about the same as in run mode.

**Comment by drozenfa** [comment]
Excellent. Thanks. And kudos for a great and useful job
